polaris-react/src/components/Choice/Choice.scss

```
@@ -4,6 +4,7 @@
   display: flex;
   justify-content: flex-start;
   padding: var(--p-space-1) 0;
+  align-items: center;
```

Does this need to be wrapped in the summer editions flag?

```suggestion
  #{$se23} & {
    align-items: center;
  }
```

---

This one was a good samaritan fix and can be applied to both se23 and the old code, but yeah good call to throw this behind the flag 👍